src/Diagnostics.ModelsAndUtils/Models/Search/SearchModels.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Diagnostics.ModelsAndUtils.Attributes;
+using Diagnostics.ModelsAndUtils.ScriptUtilities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class SearchResult
+    {
+        public string Detector { get; set; }
+
+        public float Score { get; set; }
+    }
+
+    public class SearchResults
```

Comment:
All these model classes will be used by Detector Authors or just our Runtime Host API.
If there are not getting used by developer authors, then move these models into Runtime Host Project.

This project only contains models and utils that are exposed to Detector Authors

Reply:
The model QueryUtterancesResults  is used by the DiagnosticApiResponse class of Diagnostics.ModelsAndUtilities as well as Diagnostics.RuntimeHost. Since we have project reference of ModelsAndUtilities in RuntimeHost (and not the other way round), we have to put this in ModelsAndUtilities.